kafka-client/src/main/java/dev/responsive/kafka/clients/DelegatingConsumer.java

```diff
@@ -37,7 +37,7 @@
 
 public abstract class DelegatingConsumer<K, V> implements Consumer<K, V> {
 
-  protected final Consumer<K, V> delegate;
+  private final Consumer<K, V> delegate;
```

Comment:
We were haphazardly invoking the `delegate` directly in some places, and calling the `super` method in others. I'm pretty sure we just always want to delegate to the super and keep this private to protect us from ourselves